<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoBatchRunner</class>
 <widget class="QMainWindow" name="AutoBatchRunner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoBatchRunner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="profileframe">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ProfileComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QToolButton" name="addButton">
         <property name="text">
          <string>+</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/default_icons/add.png</normaloff>:/default_icons/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="deleteButton">
         <property name="text">
          <string>-</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/default_icons/remove.png</normaloff>:/default_icons/remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="editButton">
         <property name="text">
          <string>*</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/default_icons/edit.png</normaloff>:/default_icons/edit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="FileQueueTable" name="fileTable">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
selection-background-color: rgb(144, 200, 246);
selection-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>ファイル</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>ツール</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>表示</string>
    </property>
    <addaction name="actionConsole"/>
    <addaction name="separator"/>
    <addaction name="actionAutohide"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionRun"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="BaseDockWidget" name="consoleDock">
   <property name="windowTitle">
    <string>コンソール</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ConsoleView" name="console">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(204, 204, 204);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ConsoleEdit" name="commandSender">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(204, 204, 204);
border: 0px</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>&gt; Press Enter to send command ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/default_icons/newfile.png</normaloff>:/default_icons/newfile.png</iconset>
   </property>
   <property name="text">
    <string>ファイルを開く</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>終了</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>コンソール</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>:/default_icons/settings.png</normaloff>:/default_icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>設定</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>:/default_icons/run.png</normaloff>:/default_icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/default_icons/pause.png</normaloff>:/default_icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/default_icons/stop.png</normaloff>:/default_icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionAutohide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>タイトルバーを自動的に隠す</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProfileComboBox</class>
   <extends>QComboBox</extends>
   <header>../profilecombobox/profilecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>BaseDockWidget</class>
   <extends>QDockWidget</extends>
   <header>basedockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileQueueTable</class>
   <extends>QTableWidget</extends>
   <header>../filequeuetable/filequeuetable.h</header>
  </customwidget>
  <customwidget>
   <class>ConsoleView</class>
   <extends>QTextEdit</extends>
   <header>consoleview.h</header>
  </customwidget>
  <customwidget>
   <class>ConsoleEdit</class>
   <extends>QLineEdit</extends>
   <header>consoleedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
